### DISCUSSION TITLE
[OP Stack Deployment] existed Dchain L2 (existed with traction) is looking for grants support

### DISCUSSION THREAD

**Discussion by oleksiivinogradov** [ethereum-optimism/developers#636] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hey, we just reach to grants (it was very complicated) and looks like till 1st Nov 2024 it was available. Or it possible to apply now with delay?

**Comment by hash1go** [comment]
Could you explain the background in more detail?

**Reply by oleksiivinogradov** [reply]
> Could you explain the background in more detail?


sure, here is it:

DChain - blockchain for games and studios. 

🔗 Deployment support with go-to-market strategy. With an average revenue of $85k per month from our games, 135+ strategic partnerships, and 16 committed projects, DChain unlocks endless opportunities. Benefit from our expert team's go-to-market strategy, unique liquidity system, and low transaction fees, while accessing our exclusive market tailored for crypto and blockchain projects.

Website: https://www.dchain.foundation/
Whitepaper: https://docs.openbisea.com/dexstudio/d-chain/d-chain

Marketing plan: https://docs.google.com/document/d/1MrC5Q7vMykBYs2HovCfYbAa_iY6o6C_ox4fCPR9frqg/edit?usp=sharing


Here is our testnet results https://dchaintestnet-2713017997578000-1.testnet.sagaexplorer.io/

**Reply by hash1go** [reply]
Which grant did you apply for?

**Reply by oleksiivinogradov** [reply]
> Which grant did you apply for?

I try OP Stack(superchain) but faced that there are limited to 1st Nov and i cant open new issue there anymore? Here is where i try:

https://github.com/ethereum-optimism/ecosystem-contributions/issues/261

and here is: "Application instructions
To apply for this Foundation Mission Request, please complete the form in the expandable section below and leave your response as a comment on this issue thread. Submissions will be open until Friday 1 November 2024, at which time the Foundation will review all submissions and select up to two individuals /teams to complete the work defined here"

**Reply by hash1go** [reply]
So you haven't applied for any grants yet, is that correct?

**Reply by oleksiivinogradov** [reply]
> So you haven't applied for any grants yet, is that correct?

yes, i don't

**Reply by hash1go** [reply]
What are you trying to do?
Due to lack of explanation about your situation and background, I don't fully understand.

**Reply by hash1go** [reply]
After explaining about DChain, you shared a link to the Foundation Mission regarding the framework for determining voting supply targets. What is the relationship between these two?
Since the application deadline for that Foundation Mission has already passed, you cannot apply for it.

**Reply by oleksiivinogradov** [reply]
> After explaining about DChain, you shared a link to the Foundation Mission regarding the framework for determining voting supply targets. What is the relationship between these two? Since the application deadline for that Foundation Mission has already passed, you cannot apply for it.

I guess its a main point to grants apply?

**Reply by oleksiivinogradov** [reply]
> What are you trying to do? Due to lack of explanation about your situation and background, I don't fully understand.

I try to find exact team, who is responsible for OP Stack grants. Maybe you can help?

**Reply by oleksiivinogradov** [reply]
> > What are you trying to do? Due to lack of explanation about your situation and background, I don't fully understand.
> 
> I try to find exact team, who is responsible for OP Stack grants. Maybe you can help?

or way, how to apply

**Reply by hash1go** [reply]
Please refer to this link for information about grants.
https://community.optimism.io/grant/grant-overview

**Reply by oleksiivinogradov** [reply]
> Please refer to this link for information about grants. https://community.optimism.io/grant/grant-overview

I checked it long time and can't found a way to receive grant focused on mirgation to OP Stack superchain 
But i saw a lot of in press like KROMA, Kraken e.t.c.

**Reply by hash1go** [reply]
[This](https://share.hsforms.com/1XHQ9Io_lT0-vf9nSVoUahQqoshb) seems to be the only available form regarding the Superchain.
The Superchain participation process is not managed by the community, but is carried out by the OP team.

**Reply by oleksiivinogradov** [reply]
> [This](https://share.hsforms.com/1XHQ9Io_lT0-vf9nSVoUahQqoshb) seems to be the only available form regarding the Superchain. The Superchain participation process is not managed by the community, but is carried out by the OP team.

How to reach OP team? This form is about something else - dapp on OP
